src/main/java/com/basistheory/resources/proxies/ProxiesClient.java

```diff
@@ -356,7 +355,7 @@ public void patch(String id, PatchProxyRequest request, IdempotentRequestOptions
         RequestBody body;
         try {
             body = RequestBody.create(
-                    ObjectMappers.JSON_MAPPER.writeValueAsBytes(request), MediaType.parse("application/merge-patch+json"));
+                    ObjectMappers.JSON_MAPPER.writeValueAsBytes(request), MediaTypes.APPLICATION_JSON);
```

Comment:
These need to be changed to `MediaType.parse("application/merge-patch+json"` There are several spots.